CMakeLists.txt

```diff
@@ -23,12 +24,14 @@ endif()
 
 option(BUILD_STATIC   "Build libXMP static library" ON)
 option(BUILD_SHARED   "Build libXMP shared library" ${BUILD_SHARED_DEFAULT})
+option(BUILD_LITE     "Build libXMP-lite library" OFF)
 if(NOT BUILD_STATIC AND NOT BUILD_SHARED)
     message(FATAL_ERROR "Both static and shared builds got disabled. You must enable at least one of them.")
 endif()
 
 option(LIBXMP_DISABLE_DEPACKERS     "Disable archive depackers" OFF)
 option(LIBXMP_DISABLE_PROWIZARD     "Disable ProWizard format loaders" OFF)
+option(LIBXMP_DISABLE_IT            "Disable IT format in libXMP-lite" OFF)
```

Comment:
Maybe add the flag that this depends on `BUILD_LITE`? I.e. it has no sense without the `BUILD_LITE=ON`

Reply:
Hm, can change to cmake_dependent_option

Reply:
Its description clearly states that it's for libxmp-lite, though